kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/VaultTestKmsFacade.java

```diff
@@ -15,7 +15,7 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 public class VaultTestKmsFacade extends AbstractVaultTestKmsFacade {
-    public static final DockerImageName HASHICORP_VAULT = DockerImageName.parse("hashicorp/vault:1.15");
+    public static final DockerImageName HASHICORP_VAULT = DockerImageName.parse("hashicorp/vault:1.18.3");
```

Comment:
Same justification as with Localstack - we value test stability.   Using floating tag was a mistake.